src/site/shared/containers/ImageExporterPopup/index.scss

```diff
@@ -163,3 +163,28 @@
         }
     }
 }
+
+.image-exporter-preview__button {
```

Comment:
Did we have some sort of style convention that dictated both the mix of `-` and `_` and also the double `_` between `preview` and `button`?

Reply:
this was just from before, we can have stylelint do naming stuff later